infra/helper.py

```diff
@@ -95,8 +95,8 @@ def main():
 
   check_build_parser = subparsers.add_parser(
       'check_build', help='Checks that fuzzers execute without errors.')
```

Comment:
How will people respond to the coverage build failure bugs. i think they are supposed to repro case when coverage build fails. I think even the wiki points to using check_build. @Dor1s would know.

Reply:
Ok i see

"For reproducing a coverage build failure, follow Code Coverage page to build your project and generate a code coverage report."